src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,324 @@
+package tasklist;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.Scanner;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
+import exceptions.TheOrangeRatchetCatException;
+
+public class TaskList {
+
+    /**
+     * Prints out all tasks currently in the list
+     *
+     * @param items represents the list of task
+     * @param scanner the scanner object to read next line of input
+     * @return returns the next input command by user
+     */
+    public static String checkList(List<Task> items, Scanner scanner) {
+        int index = 1;
+        StringBuilder sbr = new StringBuilder("Here are the tasks in your list: \n");
+        System.out.println("Here are the tasks in your list:");
+        for (Task item : items) {
+            System.out.println(index + "." + item);
+            sbr.append(index + "." + item + "\n");
+            index++;
+        }
+        //return scanner.nextLine();
+        return sbr.toString();
+    }
+
+    /**
+     * Marks a particular task with that specific index as done
+     *
+     * @param input represents the index of the task to mark
+     * @param items represents the list of task
+     * @param scanner the scanner object to read next line of input
+     * @return returns the next input command by user
+     */
+    public static String markingTask(String input, List<Task> items, Scanner scanner) {
```

Comment:
Would using imperative naming e.g `markTask` be more preferable in this scenario? Same for the other functions in this class.